src/Plaid.php

```diff
@@ -1014,4 +1014,21 @@ public function listPayments(array $options = []): object
 			$this->buildRequest("post", "payment_initiation/payment/list", $this->clientCredentials($params))
 		);
 	}
+
+	/**
+	 * Send a refresh request for given item.
+	 *
+	 * @param string $access_token
+	 * @return object
+	 */
+	public function requestTransactionsRefresh(string $access_token): object
```

Comment:
Can we rename this to refreshTransactions()? I have a 1.0-dev branch that includes this method which will make it easier for backwards compatibility.

Reply:
Sure I can!